src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/EmptyLineSeparatorCheck.java

```diff
@@ -411,7 +411,10 @@ public void visitToken(DetailAST ast) {
             processMultipleLinesInside(ast);
         }
         if (ast.getType() == TokenTypes.PACKAGE_DEF) {
-            checkCommentInModifiers(ast);
+            final Optional<DetailAST> comment = findCommentUnder(ast);
```

Comment:
lets keep it 

but lets remove util methods , that are already in our Utils classes
https://github.com/checkstyle/checkstyle/blob/c0290eee78318428e2b14432968de34f34ee92e0/src/main/java/com/puppycrawl/tools/checkstyle/utils/TokenUtil.java#L209
we can reuse util method

candidates:
https://github.com/checkstyle/checkstyle/blob/98d1a3edf17c42766fed1acb7c1002a682d9c8c7/src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/EmptyLineSeparatorCheck.java#L911-L925

Reply:
Done. Removed `isComment()`.